dashboard/controller/classrooms.js

```diff
@@ -74,7 +79,9 @@ exports.addClassroom = function(req, res) {
 		req.body.name = req.body.name.trim();
 		req.body.students = req.body.students || [];
 		req.body.color = JSON.parse(req.body.color);
-		req.assert('name', common.l10n.get('NameNotAlphanumeric')).isAlphanumeric();
+		if(!alphacheck(req.body.name)){
```

Comment:
Two issues:
1. localization name should not have spaces.
2. override existing function or use `matches`
https://github.com/chriso/validator.js/issues/305
3.